src/viam/examples/project/cmake/conanfile.txt

```diff
@@ -0,0 +1,6 @@
+[requires]
+viam-cpp-sdk/0.0.10
```

Comment:
Ditto re version

Reply:
Ok so this part I think we might not be able to derive the version programatically. In keeping with the spirit that stuff in the `examples/project` directory should be usable as starter code, a `conanfile.txt` is the easiest way to get started with a `conan` project that depends on the SDK.

A `conanfile.py` could in theory get the version like the root `conanfile.py` as sketched above, but then this would be grabbing a version from `../../../../CMakeLists.txt` which would probably merit an all-caps warning comment to delete that in your own project and write in an explicit version.

We could specify a version range like `viam-cpp-sdk/[>=0.0.12]` (or whichever release this lands in) but I don't love that.

I think the last option is that if the SDK release instructions say "manually bump the version in the CML.txt *and also* the example project conanfile.txt" which might not be so terrible

Reply:
Yeah, I hadn't really caught that this was in the example. I just saw a hardcoded version and the usual reflexes kicked in. I don't know: I sort of find the `viam-cpp-sdk/[>=0.0.12]` sort of intuitively appealing? Can you elaborate on why you don't love it?

Reply:
I think it might be fine actually, looking at https://docs.conan.io/2/tutorial/versioning/version_ranges.html it seems like it will prefer to use the newer one if available, just trying to imagine a situation where the user is accidentally getting an old version because they hadn't yet built a new one